packages/zosjobs/src/DownloadJobs.ts

```diff
@@ -101,8 +104,21 @@ export class DownloadJobs {
         this.log.trace("Entering downloadSpoolContentCommon with parms %s", JSON.stringify(parms));
         ImperativeExpect.keysToBeDefined(parms, ["jobFile"], "You must specify a job file on your 'parms' parameter" +
             " object to the downloadSpoolContentCommon API.");
-        const job = parms.jobFile;
 
+        //waiting for job to be active before continuing with job download
+        if (parms.waitForActive) {
```

Comment:
From my understanding, if this parameter is passed, but the job is already in output status, this will never return. Is that the intended behavior?

Reply:
the underlying function does have an order of precedence.
`Input < Active < Output` and the check is for `>=` 😋 

Following this logic path...
```
MonitorJobs.waitForActiveStatus > MonitorJobs.waitForStatusCommon > MonitorJobs.waitForStatusCommon > MonitorJobs.pollForStatus > MonitorJobs.checkStatus
```

you should end up in this check 😋 
https://github.com/zowe/zowe-cli/blob/download-jobs-enhancement/packages/zosjobs/src/MonitorJobs.ts#L281-L285

Reply:
This sequence of commands should help demonstrate that it's possible for `--wfa` to stilll finish successfully
<img width="806" alt="image" src="https://github.com/user-attachments/assets/307c48a6-27b3-45b4-88fa-cda4504ca414">